client/src/App.tsx

```diff
@@ -80,12 +82,19 @@ export default function App() {
     };
 
     return (
-
-        <Routes>
-            <Route path="/" element={<Home login={login} logout={logout} loginErrorMessage={errorMessage} error={error} setError={setError} user={user} />} />
-            <Route path="/dashboard" element={<Console user={user} />} />
-            <Route path="/documents/:id" element={<Document user={user} />} />
-            <Route path="/*" element={<NotFound />} />
-        </Routes>
+        <Layout
+            login={login}
```

Comment:
<img width="1440" alt="Screen Shot 2024-12-11 at 10 04 37" src="https://github.com/user-attachments/assets/1a9577c0-7c20-4cf4-9a84-660f954c67af">
the list has this problem after logging in. I checked on both chrome & firefox

Reply:
didnt touch the list

Reply:
I resolved it, can I push on your branch? A Question: have you changed the delete icon? because I cannot see that in the controls

Reply:
> I resolved it, can I push on your branch? A Question: have you changed the delete icon? because I cannot see that in the controls

I pushed! please check that